compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -1110,7 +1111,42 @@ static void addLowerToLLVMGPUPasses(OpPassManager &modulePassManager,
       .addPass(createConvertBf16ArithToF32Pass)
       .addPass(createConvertBf16ToUInt16BuffersPass)
       // Convert math dialect elementry functions to polynomial form.
-      .addPass(createPolynomialApproximationPass)
+      .addPass([] {
+        PolynomialApproximationPassOptions options;
```

Comment:
Bit of a handful. Maybe just add an option to the pass, where saying "all" means it will use this list.